src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+
 public class CinemaHallRequestDto {
+    @Min(value = 0)
```

Comment:
this annotation won't work until you'll add annotation @Valid to the controller method argument. fix for all dtos